src/components/Links/Links.scss

```diff
@@ -0,0 +1,10 @@
+section {
+    nav {
+        margin-top: 20px;
```

Comment:
Instead of including margin-top here, you could use Bootstrap's class `.mb-3` on the `h3` element.